Sources/EpoxyCollectionView/CollectionView/Internal/CollectionViewScrollToItemHelper.swift

```diff
@@ -214,7 +214,7 @@ final class CollectionViewScrollToItemHelper {
     case .partiallyOrFullyVisible(let frame):
       let targetOrigin = targetOriginForVisibleItem(
         withFrame: frame,
-        inVisibleBounds: collectionView.bounds,
+        inVisibleBounds: collectionView.bounds.inset(by: collectionView.adjustedContentInset),
```

Comment:
welp, such an easy fix! Thanks so much for investigating and finding the solution!